src/layers/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 file(GLOB_RECURSE layers_src *.cpp)
 add_library(layers_lib STATIC "${LAYERS_HEADERS}" "${layers_src}")
+
+target_include_directories(layers_lib PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}/../include
+)
+
```

Comment:
I'm not really sure this code makes any sense. I think `src/include` path does not exist